README.md

```
@@ -48,6 +48,21 @@ bin/kamal deploy
 
 ## 開発メモ
 
+## 開発サーバーの立ち上げ方
+
+開発サーバーを立ち上げるには、以下のコマンドを実行します：
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)